### PR TITLE
Most of the NPC text in Gateway of the Great King done by RTT

### PR DESCRIPTION
**Pull request by Ragunaga**
Single line in Quest from the area, so went ahead and included that file aswell.

Had to do a bit of retranslation in some areas, some lines have large changes.

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -147,18 +147,18 @@ QUEST_LV_0100_20150317_000146	$Sir, get a hold of yourself. Sir!
...
 QUEST_LV_0100_20150317_000151	$The entrance to the Royal Mausoleum is hidden within this gigantic canyon somewhere.{nl}The start of the canyon begins here.
 QUEST_LV_0100_20150317_000152	$Many experts have come here to conduct research on the Royal Mausoleum.{nl}Maybe we can find some clues about the Medis Diena here.
 QUEST_LV_0100_20150317_000153	$Why are there so many monsters in the canyon when there is so little plant life around?
 QUEST_LV_0100_20150317_000154	$What happened to my request? Is it not done yet?
 QUEST_LV_0100_20150317_000155	$Look closely at the ground.{nl}It may have fallen down.
-QUEST_LV_0100_20150317_000156	$Hmm, only these ones?{nl}Well, thank you for finding this much at least.{nl}I can just copy again from the beginning.
-QUEST_LV_0100_20150317_000157	$Niels sent you too?{nl}I am not going. I have to research the wild flowers.{nl}
-QUEST_LV_0100_20150317_000158	$I guess what I saw was an illusion.{nl}I must reward you for protecting a mineralogist like me.{nl}
+QUEST_LV_0100_20150317_000156	$Hmm, only these ones?{nl}Well, thank you for finding this much at least.{nl}I can just copy the others from the beggining again.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Beggining -> beginning

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -1479,11 +1479,11 @@ QUEST_LV_0100_20150317_001478	The space may exist anywhere, but it may also not
...
-QUEST_LV_0100_20150317_001482	Our family has been waiting for you for a long time.{nl}With this igniter, please light the signal at the top.
-QUEST_LV_0100_20150317_001483	Since you, the selected person, has come, it is sending us a signal to greet you.{nl}After you light the signal, tell Niels that I am sorry.
+QUEST_LV_0100_20150317_001482	$Our family has been waiting for your appearance for a long time.{nl}With this igniter, please light the signal at the top.
+QUEST_LV_0100_20150317_001483	$Since you have appeared, it is sending us a signal to greet you.{nl}After you light the signal, tell Niels that I am sorry.
 QUEST_LV_0100_20150317_001484	There's nothing that I can't get.{nl}If you could give me some money.
-QUEST_LV_0100_20150317_001485	Do you need anything?{nl}I can get you anything if you pay me enough.{nl}But, it will be expensive.
-QUEST_LV_0100_20150317_001486	Don't ever think about going yourself without mercenaries.{nl}There are many monsters out there that can't even be defeated by mercenaries.
+QUEST_LV_0100_20150317_001485	$Do you need anything?{nl}I can get you anything aslong as you pay me enough.{nl}But, it will be expensive.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Aslong -> as long

**Comment by Ragunaga** [comment]
Fixed all~

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Ragunaga** [comment]
I know there are 2 conflicts at lines 6867 and 6868. I am unsure of how to fix it due to a problem with git pull. I do believe my lines to be better however.

EDIT: NVM, FIXED.

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**